en/docs/reference/connectors/msazuredatalakestorage-connector/microsoft-azure-data-lake-storage-connector-reference.md

```diff
@@ -0,0 +1,1583 @@
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>flush</td>
+            <td>This request will succeed only if the provided leaseId matches the actual lease on the resource.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>leaseId</td>
+            <td>The operation will be performed only if the resource has not been modified since the specified time.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>leaseAction</td>
+            <td>The operation will be performed only if the resource's ETag matches the specified ETag value.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>leaseDuration</td>
+            <td>The operation will be performed only if the resource has been modified since the specified time.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>proposedLeaseId</td>
+            <td>A proposed lease ID that can be set when acquiring or changing a lease.</td>
+            <td>No</td>
+        </tr>
+    </table>
+
+    **Sample configuration**
+
+    ```xml
+    <msazuredatalakestorage.appendFile configKey="con1">
+        <fileSystemName>{${payload.fileSystemName}}</fileSystemName>
+        <filePathToAppend>{${payload.filePathToAppend}}</filePathToAppend>
+        <inputType>Local File</inputType>
+        <localFilePath>{${payload.localFilePath}}</localFilePath>
+        <timeout></timeout>
+        <responseVariable>msazuredatalakestorage_appendFile_1</responseVariable>
+        <overwriteBody>true</overwriteBody>
+        <flush>true</flush>
+        <leaseId></leaseId>
+        <leaseAction>None</leaseAction>
+        <leaseDuration></leaseDuration>
+        <proposedLeaseId></proposedLeaseId>
+    </msazuredatalakestorage.appendFile>
+    ```
+    
+    **Sample request**
+
+    ```json
+    {
+        "fileSystemName": "sales",
+        "filePathToAppend": "section1/marketing/sample.json",
+        "localFilePath": "/path/to/local/file.txt",
+    }
+    ```
+
+??? note "getMetadata"
+    The getMetadata operation retrives metadata of a file. See the [related API documentation](https://learn.microsoft.com/en-us/java/api/overview/azure/storage-file-datalake-readme?view=azure-java-stable) for more information.
+    <table>
+        <tr>
+            <th>Parameter Name</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+        <tr>
+            <td>fileSystemName</td>
+            <td>The name of the file system.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>filePath</td>
+            <td>The path of the file.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>responseVariable</td>
+            <td>Response variable name</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>overwriteBody</td>
+            <td>Replace the Message Body in Message Context with the response of the operation.</td>
```

Comment:
```suggestion
            <td>Replace the Message Body in Message Context with the output of the operation (This will remove the payload from the above variable).</td>
```